src/mqt/bench/utils.py

```diff
@@ -342,3 +343,74 @@ def postprocess_single_oqc_file(filename: str):
 
 def create_zip_file():
     return subprocess.call(f"zip -rj {get_zip_file_path()} {get_qasm_output_path()}", shell=True)
+
+
+def calc_qubit_index(qargs: list[Any], qregs: list[QuantumRegister], index: int) -> int:
+    offset = 0
+    for reg in qregs:
+        if qargs[index] not in reg:
+            offset += reg.size
+        else:
+            qubit_index = offset + reg.index(qargs[index])
+            return qubit_index
+    error_msg = "Global qubit index for given local qubit index not found."
```

Comment:
While this error message is better than the message before, it still contains absolutely no information about the local qubit index that was the cause of the error.
```suggestion
    error_msg = f"Global qubit index for local qubit {index} index not found."
```